scripts/linux/runE2ETest.sh

```diff
@@ -102,7 +102,7 @@ function get_long_haul_deployment_artifact_file() {
     local nestedEdgeTest=$(printenv E2E_nestedEdgeTest)
 
     if [[ ! -z "$nestedEdgeTest" ]]; then
-      path="$E2E_TEST_DIR/artifacts/core-linux/e2e_deployment_files/nested_long_haul_deployment.template.json"
+      path="$E2E_TEST_DIR/artifacts/core-linux/e2e_deployment_files/nestededge_bottomLayerBaseDeployment_long_haul_amqp.template.json"
```

Comment:
name convention: `neste_dedge_bottom_layer_base_deployment_long_haul_amqp`?

Reply:
I was following how the other deployment files did it - nestededge_middleLayerBaseDeployment.json